src/classes/RD2_Constants.cls

```diff
@@ -45,6 +44,15 @@ public inherited sharing class RD2_Constants {
         No_Action
     }
 
+    /**
+    * @description Enum to specify behavior of how to handle installment opportunity auto-creation
+    */
+    public enum InstallmentCreateOptions {
+        Always_Create_Next_Installment,
+        Disable_First_Installment_on_Create,
```

Comment:
Let's keep the API name as `Disable_First_Installment`. The clarification is required for label only, ie user facing message.